hypha/apply/api/v1/filters.py

```diff
@@ -57,6 +62,11 @@ def filter_active(self, qs, name, value):
         else:
             return qs.inactive()
 
+    def filter_id(self, qs, name, value):
+        if not value:
+            return qs
+        return qs.filter(id__in=self.request.GET.getlist('id'))
```

Comment:
@vimal1083, shouldn't this depend on `value`? `value` passed for the filter is not being used, is that correct?